src/server/game/Handlers/ChatHandler.cpp

```diff
@@ -384,7 +384,7 @@ void WorldSession::HandleChatMessage(ChatMsg type, uint32 lang, std::string msg,
             if (Channel* chn = ChannelMgr::GetChannelForPlayerByNamePart(target, sender))
             {
                 sScriptMgr->OnPlayerChat(sender, type, lang, msg, chn);
-                chn->Say(sender->GetGUID(), msg.c_str(), lang, nullptr);
+                chn->Say(sender->GetGUID(), msg.c_str(), lang, "");
```

Comment:
"" is new default value, so it can be omitted
chn->Say(sender->GetGUID(), msg.c_str(), lang); should work